README.md

```diff
@@ -25,59 +25,68 @@ The `ovos_config` package determines which config files to load based on `ovos.c
 
 A simple `ovos_config` should have a structure like:
 
-```yaml
-xdg: True
-base_folder: mycroft
-config_filename: mycroft.conf
-default_config_path: <Path to Installed Core>/configuration/mycroft.conf
-module_overrides: {}
-submodule_mappings: {}
+```json
+{
+"base_folder": "mycroft",
+"config_filename": "mycroft.conf",
+"default_config_path": "<Path to Installed Core>/configuration/mycroft.conf",
+"module_overrides": {},
+"submodule_mappings": {}
+}
 ```
 
 Non-Mycroft modules may specify alternate config paths. A call to `get_ovos_config` from 
 `neon_core` or `neon_messagebus` will return a configuration like:
 
-```yaml
-xdg: True
-base_folder: neon
-config_filename: neon.yaml
-default_config_path: /etc/example/config/neon.yaml
-module_overrides:
-  neon_core:
-    xdg: true
-    base_folder: neon
-    config_filename: neon.yaml
-    default_config_path: /etc/example/config/neon.yaml
-submodule_mappings:
-  neon_core: neon_core
-  neon_core.skills.skill_manager: neon_core
-  neon_messagebus: neon_core
-  neon_speech: neon_core
-  neon_audio: neon_core
-  neon_gui: neon_core
+```json
+{
+  "xdg": true,
+  "base_folder": "neon",
+  "config_filename": "neon.yaml",
+  "default_config_path": "/etc/example/config/neon.yaml",
+  "module_overrides": {
+    "neon_core": {
+      "base_folder": "neon",
+      "config_filename": "neon.yaml",
+      "default_config_path": "/etc/example/config/neon.yaml"
+    }
+  },
+  "submodule_mappings": {
+    "neon_core": "neon_core",
```

Comment:
this one should not be here since it maps to itself

Reply:
Right; I have it in Neon configs just because of how I auto-populate submodules but it is extraneous